### PR TITLE
Allow SCM users to request cases for a judge

### PR DESCRIPTION
**Pull request by hschallhorn**
Final PR of stack to resolve first AC of #12424

Part 1: #13727
Part 2: #13729
Part 3: #13731

### Description
Quick refactor to pull distribution button out into its own component.

### Acceptance Criteria
- [ ] No tests fail!

### Testing plan
- [ ] Ensure master and this branch match

### UI changes
None!

Before|After
---|---
Screen Shot 2020-03-17 at 3 39 36 PM|Screen Shot 2020-03-17 at 3 39 07 PM

## OVERALL CHANGE
ezgif-5-e08c41975fcd

### Description
Allows SCM members to request cases for judges

### Acceptance Criteria
- [ ] SCM team member can request cases for a judge

### Testing plan
1. Log in as an SCM team member
2. Navigate to BVAAABSHIRE's assign queue at /queue/3/assign
3. Select "Request more case"
4. Ensure no errors
5. Ensure cases are distributed

### UI Changes

- [ ] SCM distributions don't throw errors

Before|After
---|---
Screen Shot 2020-03-17 at 3 48 27 PM|Screen Shot 2020-03-17 at 4 52 54 PM

### CODE REVIEW

File: client/app/queue/UnassignedCasesPage.jsx [hunk 1]
```diff
@@ -21,6 +22,13 @@ import { css } from 'glamor';
 
 const assignSectionStyling = css({ marginTop: '30px' });
 const loadingContainerStyling = css({ marginTop: '-2em' });
+const assignAndRequestStyling = css({
+  display: 'flex',
+  alignItems: 'center',
+  flexWrap: 'wrap',
+  '& > *': { marginRight: '1rem',
+    marginTop: '0',
+    marginBottom: '16px' } });
```
Comment (hschallhorn) [inline client/app/queue/UnassignedCasesPage.jsx]: Pulled from assign widget. Keeps the widget to the left and the request cases button to the right.

File: client/app/queue/UnassignedCasesPage.jsx [hunk 2]
```diff
@@ -8,6 +8,7 @@ import {
   initialAssignTasksToUser
 } from './QueueActions';
 import AssignWidget from './components/AssignWidget';
+import RequestDistributionButton from './components/RequestDistributionButton';
```
Comment (hschallhorn) [inline client/app/queue/UnassignedCasesPage.jsx]: Bet to view these changes ignoring whitespace!

File: client/app/queue/UnassignedCasesPage.jsx [hunk 3]
```diff
@@ -37,12 +45,15 @@ class UnassignedCasesPage extends React.PureComponent {
...
-            showRequestCasesButton />
+          <div {...assignAndRequestStyling}>
+            <AssignWidget
+              userId={userId}
+              previousAssigneeId={userId}
+              onTaskAssignment={this.props.initialAssignTasksToUser}
+              selectedTasks={selectedTasks}
+              showRequestCasesButton />
+            <RequestDistributionButton userId={userId} />
+          </div>
```
Comment (hschallhorn) [inline client/app/queue/UnassignedCasesPage.jsx]: Woo main change! No clue why these were ever included together in the same component.

File: client/app/queue/components/AssignWidget.jsx [hunk 4]
```diff
@@ -14,7 +14,6 @@ import {
   setSelectedAssignee,
   setSelectedAssigneeSecondary
 } from '../uiReducer/uiActions';
-import { requestDistribution } from '../QueueActions';
```
Comment (hschallhorn) [inline client/app/queue/components/AssignWidget.jsx]: Basically pulled anything related to distribution out into a new component.

File: client/app/queue/UnassignedCasesPage.jsx (hunk 3 above)
Comment (ajspotts) [reply client/app/queue/UnassignedCasesPage.jsx]: good question either way nice refactor!

**Pull request merged**